src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,288 @@
+import * as React from "react";
+import {
+  AppBar,
+  Box,
+  CssBaseline,
+  Divider,
+  Drawer,
+  IconButton,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  Switch,
+  Toolbar,
+  FormGroup,
+  FormControlLabel,
+  Collapse,
+} from "@mui/material";
+import PropTypes from "prop-types";
+import MenuIcon from "@mui/icons-material/Menu";
+import HomeIcon from "@mui/icons-material/Home";
+import HistoryIcon from "@mui/icons-material/History";
+import StarIcon from "@mui/icons-material/Star";
+import SettingsIcon from "@mui/icons-material/Settings";
+import AccountCircleIcon from "@mui/icons-material/AccountCircle";
+import LogoutIcon from "@mui/icons-material/Logout";
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import DoNotDisturbOnIcon from "@mui/icons-material/DoNotDisturbOn";
+import TimerIcon from "@mui/icons-material/Timer";
+import { grey, green, red, yellow } from "@mui/material/colors";
+import { ExpandLess, ExpandMore} from "@mui/icons-material";
+import { TabStyled } from "./style";
+
+const drawerWidth = 240;
+
+function ResponsiveDrawer(props) {
+  const { window } = props;
+  const [mobileOpen, setMobileOpen] = React.useState(false);
+  const [currentWindow, setCurrentWindow] = React.useState(null);
+  const [openDropDown, setOpenDropDown] = React.useState(false);
+  const [openSwitch, setOpenSwitch] = React.useState(true);
+  const [busySwitch, setBusySwitch] = React.useState(true);
+  const [notification, setNotification] = React.useState({});
+
+  const iconList = [
```

Comment:
It would be a good idea to have a seperate file for this iconList.
Maybe call it as drawerSections.js